charms/worker/k8s/lib/charms/k8s/v0/k8sd_api_manager.py

```diff
@@ -372,20 +372,38 @@ def _send_request(
         except (socket.error, HTTPException) as e:
             raise K8sdConnectionError(f"HTTP or Socket error: {e}") from e
 
-    def create_join_token(self, name: str):
+    def create_join_token(self, name: str, worker: bool = False):
```

Comment:
@mateoflorido correct me if i'm wrong but when you update a library don't we have to rev it or something?

https://discourse.charmhub.io/t/topic/5781#heading--update-a-charm-library